src/components/spaces/populations/populations-container.tsx

```diff
@@ -30,6 +30,13 @@ export class PopulationsComponent extends BaseComponent<IProps, IState> {
     super(props);
   }
 
+  public componentWillUnmount() {
+    const populations = this.props.stores && this.props.stores.populations;
```

Comment:
I wouldn't have thought to include null checks here, were there problems without them?

Reply:
Simplified in 6754961d8cc. Thanks for noticing! (Still confused, though.)